lychee-bin/src/main.rs

```diff
@@ -288,7 +288,13 @@ fn underlying_io_error_kind(error: &Error) -> Option<io::ErrorKind> {
 async fn run(opts: &LycheeOptions) -> Result<i32> {
     let inputs = opts.inputs()?;
 
-    let mut collector = Collector::new(opts.config.base.clone())
+    if let Some(root_dir) = &opts.config.root_dir {
+        if root_dir.is_relative() {
+            bail!("`--root_dir` must be an absolute path");
```

Comment:
One last thing; I think we should move this to `Collector::new`. lychee can also be used as library through lychee-lib so that's another entrypoint where this invariant should be enforced

Reply:
Good idea.

Reply:
done